src/components/DatePicker.js

```diff
@@ -308,122 +229,82 @@ const DatePicker = React.createClass({
         boxSizing: 'border-box',
         display: this.state.showCalendar ? 'block' : 'none',
         marginTop: 10,
-        padding: '20px 20px 0 20px',
+        padding: 20,
         position: 'absolute',
         right: 0,
         width: 287,
         zIndex: 10
       },
-      caret: {
-        fill: this.state.showCalendar ? this.props.primaryColor : StyleConstants.Colors.ASH,
-        position: 'absolute',
-        right: 5,
-        top: '50%',
-        transform: 'translateY(-50%)'
-      },
-      caretWrapper: {
-        position: 'absolute',
-        top: '50%',
-        right: 5
-      },
-      clearFix: {
-        clear: 'both',
-        marginBottom: 15
-      },
-      component: Object.assign({
-        backgroundColor: StyleConstants.Colors.WHITE,
-        clear: 'both',
-        color: StyleConstants.Colors.BLACK,
-        display: 'inline-block',
-        float: 'right',
-        fontFamily: StyleConstants.FontFamily,
-        fontSize: StyleConstants.FontSizes.MEDIUM,
+
+      //Calendar Header
+      calendarHeader: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        color: StyleConstants.Colors.CHARCOAL,
+        fontSize: StyleConstants.FontSizes.LARGE,
         marginBottom: 15,
+        height: 30,
         position: 'relative',
-        WebkitAppearance: 'none',
-        width: '100%',
+        textAlign: 'center'
+      },
```

Comment:
The left and right carets aren't quite in-line with the week headers. I found that adding `padding: 0 5px` here fixed the issue but maybe there's a sexier way to do it.